src/app/chat/views/messages.nim

```diff
@@ -101,6 +101,24 @@ QtObject:
 
     return updatedMessage
 
+  proc userNameOrAlias*(self: MessageView, pubKey: string): string {.slot.} =
```

Comment:
this function already exists in mutliple place, could we keep only one? At least do not add another one ;)

Reply:
Yes already done. Realised after raising the review, I forgot to use it

Reply:
Actually let me see if I can put it in one place too :)

Reply:
not doing to right, but I have reused the api from message_format, seemed like the right place for the API to be